index.js

```diff
@@ -0,0 +1,13 @@
+var express = require( "express" );
+var app, server, port;
```

Comment:
We had [reasons](https://github.com/jquery/contribute.jquery.org/pull/105#issue-56683932) to remove it from the style guide so let's not do it anymore too :wink: